WordPress/src/main/java/org/wordpress/android/ui/notifications/utils/NotificationsUtils.java

```diff
@@ -85,11 +84,6 @@ public static void getPushNotificationSettings(Context context, RestRequest.List
         if (!AccountHelper.isSignedInWordPressDotCom()) {
             return;
         }
-
-        if (TextUtils.isEmpty(GCMRegistrar.getRegistrationId(context))) {
-            AppLog.e(T.NOTIFS, "can't get push notification settings, gcm token is null.");
-        }
-
```

Comment:
@roundhill I removed the Log above cause we can't test synchronously for a Registration Id anymore. But we'll still have error messages in logs if Play Services are not available or if gcm token is not ready (added in 5febc66259e7de56020f208d59f7ada3641458d7)